index.html

```diff
@@ -0,0 +1,264 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <title>First Task</title>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
```

Comment:
Now no media queries will be working, guess why? :) tip: you're missing a meta tag